images/sles-sap/content.yaml

```diff
@@ -14,6 +14,14 @@ image:
           requires:
             - _attributes:
                 profile: azure-base
+    _namespace_profiles_standard:
```

Comment:
This overwrites the standard EC2 and GCE profiles. Since the GCE-3P profile is just an an add-on, it should be in its own namespace, e.g. `_namespace_profiles_gce_3p` or `_namespace_profiles_gce_extra`.